tox.ini

```diff
@@ -19,12 +20,14 @@ deps =
        -rrequirements/requirements-optionals.txt
 
 [testenv:py27-flake8]
+downloadcache={env:PIP_DOWNLOAD_CACHE}
```

Comment:
This is a bit odd as tox is actually using [``PIP_DOWNLOAD_CACHE`` overwrites the ``downloadcache``](http://tox.readthedocs.org/en/latest/config.html#confval-downloadcache=path) value if it is defined, but [``downloadcache`` must be present in the first place](https://bitbucket.org/hpk42/tox/src/9d3605e2389a042f246b8720bdf7fdcaa9ff0a17/tox/_venv.py?at=default#cl-260), so ``downloadcache=true`` may be more appriopriate, as the value will be ignored anyway, but this is more explicit.